### PR TITLE
Add Windows Server Core 20H2 helix Amd64

### PR DESCRIPTION
**Pull request by aik-jahoda**
Adds Windows Server Core 20H2 Amd64 for the Helix.

This is copy paste of  Windows Server Core ltsc2019 image with change of base image.

Part of #502 

/cc @wfurt

### CODE REVIEW

File: src/windowsservercore/20h2/helix/amd64/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        Set-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps\dotnet.exe' -Value 'C:\cores' -Name DumpFolder -Force ; `
+        Set-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps\dotnet.exe' -Value 2 -Name DumpCount -Force ; `
+        New-Item -Path 'HKLM:\SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps\' -Name 'corerun.exe' -Force -ErrorAction SilentlyContinue ; `
+        Set-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps\corerun.exe' -Value 2 -Name DumpType -Force ; `
+        Set-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps\corerun.exe' -Value 'C:\cores' -Name DumpFolder -Force ; `
+        Set-ItemProperty -Path 'HKLM:\SOFTWARE\Microsoft\Windows\Windows Error Reporting\LocalDumps\corerun.exe' -Value 2 -Name DumpCount -Force && `
+   setx /M PATH "%PATH%;C:\Python;C:\python\scripts"    
+
+WORKDIR C:\\Work
+
```
Comment (mthalman) [inline src/windowsservercore/20h2/helix/amd64/Dockerfile:33]: Remove extra line.

**Review by mthalman — COMMENTED** [review]
This requires adding new pipeline YAML files.  You can copy the existing https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/main/eng/pipelines/dotnet-buildtools-prereqs-windowsservercore-2004-pr.yml and https://github.com/dotnet/dotnet-buildtools-prereqs-docker/blob/main/eng/pipelines/dotnet-buildtools-prereqs-windowsservercore-2004.yml files. Then find-and-replace "2004" with "20H2".

**Comment by aik-jahoda** [comment]
Pipelines added.

**Review by mthalman — APPROVED** [review]

**Comment by mthalman** [comment]
Build is done.  Published image metadata: https://github.com/dotnet/versions/commit/b12b18a9b113a0df27fa16fa23d39fbeacaeb07d#diff-abb3fd8a62995aae16ec0588e0132a171ce9363b95319c4845f4a5cf7b34d91c

**Pull request merged**